pkg/serializer/internal/metrics/service_checks_test.go

```diff
@@ -126,7 +126,7 @@ func TestPayloadsEmptyServiceCheck(t *testing.T) {
 
 func TestPayloadsServiceChecks(t *testing.T) {
 	config := mock.New(t)
-	config.Set("serializer_max_payload_size", 200, pkgconfigmodel.SourceAgentRuntime)
+	config.Set("serializer_max_payload_size", 250, pkgconfigmodel.SourceAgentRuntime)
```

Comment:
The max compress bound for `zstd` is larger than for `zlib`. So to get this test to pass I have had to increase this size. It does indicate that a side effect of moving to `zstd` will be less events per payload.

When looking at the actual payload sizes in this test we have:

With Zstd:
compressed size: 108
compressed size: 108
compressed size: 92
 
With Zlib:
compressed size: 116
compressed size: 113
compressed size: 96